### PR TITLE
qemu: update to 7.2.0

### PR DESCRIPTION
**Pull request by vooon**
Signed-off-by: Vladimir Ermakov <vooon341@gmail.com>

Maintainer: @yousong
Compile tested: x86_64, OpenWrt master (https://github.com/openwrt/openwrt/commit/eaf5c85fd3d7d8f35dd80e6816c79dc35016a83f)
Run tested: x86_64 host, KVM for qemu-x86_64-softmmu + Bios VM.

Description:
- Version updated to 7.2.0
- Outdated patches to build system was removed

### CODE REVIEW

File: utils/qemu/patches/0003-configure-enable-guest_agent-no-matter-whether-softm.patch [hunk 1]
```diff
@@ -1,25 +0,0 @@
-From 3f07c9cb96b361f07ce637088f818bbe0edbcde3 Mon Sep 17 00:00:00 2001
-From: Yousong Zhou <yszhou4tech@gmail.com>
-Date: Fri, 7 Feb 2020 03:02:44 +0800
-Subject: [PATCH] configure: enable guest_agent no matter whether softmmu is
- enabled
-
-guest_agent as a tool to be run on guest machines does not depend on
-whether there is a softmmu is to be built at this configure/make run
-
-Fixes a512590 ("configure: qemu-ga is only needed with softmmu targets")
```
Comment (yousong) [inline utils/qemu/patches/0003-configure-enable-guest_agent-no-matter-whether-softm.patch:10]: Has the upstream changed the build system behavior that this patch is not needed anymore?

Comment (vooon) [reply utils/qemu/patches/0003-configure-enable-guest_agent-no-matter-whether-softm.patch:10]: They do more in meson, configure script slightly changed. But i'm not yet fully tested new makefile.

**Comment by BKPepe** [comment]
What's happening with this? Is it fully tested, which was blocking to merge this? Would it be possible to rebase it on top of the master branch?

**Comment by vooon** [comment]
@BKPepe i didn't have enough time to complete. Now probably need to update to 7.2.0.

Maybe i will have some time for this. Almost forgot about that PR.

**Comment by vooon** [comment]
Ping @neheb , @BKPepe , @yousong .

I've tested on x86_64 with bios VM.
I've run Guest Agent inside my VPN node, but it doesn't really do anything, same as before :).

File: utils/qemu/Makefile [hunk 2]
```diff
@@ -311,11 +311,9 @@ CONFIGURE_ARGS +=			\
 	--enable-vhost-crypto		\
 	--enable-vhost-kernel		\
 	--enable-vhost-net		\
-	--enable-vhost-scsi		\
 	--enable-vhost-user		\
-	--enable-vhost-user-fs		\
 	--enable-vhost-user-blk-server	\
-	--enable-vhost-vsock		\
+	--enable-vduse-blk-export	\
```
Comment (yousong) [inline utils/qemu/Makefile:310]: Consider making it default off if the feature is not that essential and widely used for now.

Comment (vooon) [reply utils/qemu/Makefile:310]: Probably, unsure where VDUSE really working yet.

**Comment by yousong** [comment]
Hi @vooon Thank you for the time and efforts.  The commits were squashed and pushed a moment ago.

**Pull request closed without merging**